bot/exts/moderation/infraction/management.py

```diff
@@ -150,6 +150,10 @@ async def infraction_edit(
             request_data['expires_at'] = None
             confirm_messages.append("marked as permanent")
         elif duration is not None:
+            now_datetime = datetime.now(duration.tzinfo) if duration.tzinfo else datetime.utcnow()
```

Comment:
Shouldn't the duration timezone always be UTC?

Reply:
I'm not sure. I copied that line from the [scheduler](https://github.com/python-discord/bot/blob/main/bot/utils/scheduling.py#L73).

Reply:
The duration timezone is always UTC. The `Expiry` typehint defines `ISODateTime` and `Duration`, which both convert their respective inputs into datetime objects within the UTC+0 timezone.

[ISODateTime](https://github.com/python-discord/bot/blob/3824ddbddff8c3191632c0479f0f594985a55b32/bot/converters.py#L425-L432)
[Duration](https://github.com/python-discord/bot/blob/3824ddbddff8c3191632c0479f0f594985a55b32/bot/converters.py#L348)
[Expiry](https://github.com/python-discord/bot/blob/3824ddbddff8c3191632c0479f0f594985a55b32/bot/converters.py#L596)

Reply:
Then ` now_datetime = datetime.utcnow()` should be enough, if I follow correctly?

Reply:
It should be enough. An alternative would be `now_datetime = datetime.now(timezone.utc)`, as the `datetime` module says that [`datetime.now(timezone.utc)` is preferred over `datetime.utcnow()`](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow) when it comes to creating `datetime` objects that represent UTC times.